src/shared/components/paginationControls/ButtonControls.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
```

Comment:
Only comment here is maybe we can change the name of `ButtonControls` to `ShowMoreButton`